code/game/objects/items/devices/scanners.dm

```diff
@@ -132,9 +132,14 @@ REAGENT SCANNER
 	var/BR = H.getBruteLoss() > 50 	? 	"<b>[H.getBruteLoss()]</b>" 	: H.getBruteLoss()
 	if(HAS_TRAIT(H, TRAIT_FAKEDEATH))
 		OX = fake_oxy > 50 			? 	"<b>[fake_oxy]</b>" 			: fake_oxy
-		to_chat(user, "<span class='notice'>Analyzing Results for [H]:\n\t Overall Status: dead</span>")
+		to_chat(user, "<span class='notice'>Analyzing Results for [H]:\n\t Overall Status: Dead</span>")
 	else
-		to_chat(user, "<span class='notice'>Analyzing Results for [H]:\n\t Overall Status: [H.stat > 1 ? "dead" : "[H.health]% healthy"]</span>")
+		var/mob/dead/observer/ghost = H.get_ghost(TRUE)
+		if(ghost && !ghost.can_reenter_corpse)
```

Comment:
I feel like this can be rewritten so it just uses 1 `to_chat`. Should save another dev a headache in the future